packages/perseus/src/widgets/free-response/free-response.stories.tsx

```diff
@@ -36,3 +36,13 @@ export const UnlimitedCharacters: Story = {
         question: "What is the theme of the essay?",
     },
 };
+
+export const QuestionWithKatex: Story = {
```

Comment:
KaTeX isn't used anymore. We've been settling into using simply `TeX` to refer to math markup.